querybook/webapp/lib/sql-helper/sql-formatter.ts

```diff
@@ -169,3 +169,15 @@ function getLanguageForSqlFormatter(language: string): SqlFormatterLanguage {
 
     return 'sql';
 }
+
+export const getQueryStatements = (query: string) => {
```

Comment:
you need to parse the query to extract statements.
This doesn't work for:
"""
alter table FOOBAR add column `SELECT` varchar(255); 
"""
or 
"""
SELECT * from TABLE where `UPDATED_AT` = 123;
"""